numpy/_core/tests/test_ufunc.py

```diff
@@ -616,6 +616,31 @@ def call_ufunc(arr, **kwargs):
         expected = call_ufunc(arr.astype(np.float64))  # upcast
         assert_array_equal(expected, res)
 
+    @pytest.mark.parametrize("ufunc", [np.add, np.equal])
+    def test_cast_safety_scalar(self, ufunc):
+        # We test add and equal, because equal has special scalar handling
+        # Note that the "equiv" casting behavior should maybe be considered
+        # a current implementation.
+        with pytest.raises(TypeError):
+            # The loop picked is integral, which is not safe
```

Comment:
```suggestion
            # this picks an integer loop, which is not safe
```

integral makes me think of calculus so let's just not use that word if it confused me reading this